Vagrantfile

```diff
@@ -227,6 +227,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       v.memory            = vagrant_openshift_config['memory'].to_i
       v.cpus              = vagrant_openshift_config['cpus'].to_i
       v.customize ["modifyvm", :id, "--cpus", vagrant_openshift_config['cpus'].to_s]
+      full_provision(override.vm)
```

Comment:
We already have centos7.  Just change os on origin-init.

I am updating the images now (still fedora 21 but running all the updates).  Let's see what shape we are in after that.